app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -651,6 +651,38 @@ public UploadResult uploadFile(String filename,
 
     }
 
+    /**
+     * This takes userName as input, which is then used to fetch the no of images deleted
+     * using OkHttp and JavaRx. This function return JSONObject
+     * @param userName
+     * @return
+     */
+    @NonNull
+    @Override
+    public Single<JSONObject> getRevertCount(String userName){
+        final String fetchRevertCountUrlTemplate =
+                wikiMediaToolforgeUrl + "urbanecmbot/commonsmisc/feedback.py";
+        return Single.fromCallable(() -> {
+            String url = String.format(
+                    Locale.ENGLISH,
+                    fetchRevertCountUrlTemplate,
+                    new PageTitle(userName).getText());
+            String userNameCaps = Character.toString(userName.charAt(0)).toUpperCase()+userName.substring(1);
```

Comment:
Is this actually required? MediaWiki usernames always start with a capital letter. Our shared prefs should already have username in this format. 

https://en.wikipedia.org/wiki/Wikipedia:Naming_conventions_(technical_restrictions)#Lowercase_first_letter